### ISSUE TITLE
Getting subwebs of the Site collection returns 403 error

### ISSUE THREAD

**Issue by djmrky** [pnp/pnpjs#2611]
### Category
- [ ] Enhancement
- [ ] Bug
- [X ] Question
- [ ] Documentation gap/issue

### Version
3.0.10
Sharepoint Online

### Expected / Desired Behavior / Question
I have a structure of sites:
/sites/projects (**Site collection**)
/sites/projects/project1 (**subsite**)
/sites/projects/project2 (**subsite**)

When I try to get the subsites of "projects" with the user who is the Owner, everything works fine (...web.webs.get())
When I do it with another user, I get error 403 even though this user has Read permissions on both the Site collection and the subsite and can access them normally. Is the Read permission not enough to list the subsites?

Is this by design or am I doing something wrong?

Thanks in advance

**Comment by djmrky** [comment]
Yeah, but is it possible to add this permission in **Sharepoint Online**? There's no option for that in the GUI.
Can it be added programmatically?

**Comment by bcameron1231** [comment]
You are able to set this option through the GUI.

Site settings ->  Site Permissions -> Permission Levels (in the Ribbon) -> Read ->Add Browse Directories under site permissions

**Comment by djmrky** [comment]
Oh, yeah, I totally forgot about that :-)

Thanks a lot Beau :-)
